packages/babel-plugin/src/css-prop/__tests__/jsx-pragma.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { transformSync } from '@babel/core';
+
+import babelPlugin from '../../index';
+
+const transform = (code: string) => {
+  return transformSync(code, {
+    configFile: false,
+    babelrc: false,
+    plugins: [babelPlugin],
+  })?.code;
+};
+
+describe('local jsx namespace', () => {
+  it('should transform css prop using jsx import source', () => {
+    const result = transform(`
+      /** @jsxImportSource @compiled/react */
+
+      <div css={{ color: 'red' }}>hello</div>;
+    `);
+
+    expect(result).toMatchInlineSnapshot(`
+      "/* File generated by @compiled/babel-plugin v0.0.0 */
+
+      import { ax, ix, CC, CS } from \\"@compiled/react/runtime\\";
+      const _ = \\"._syaz5scu{color:red}\\";
+
+      /** @jsxImportSource @compiled/react */
+      <CC>
+          <CS>{[_]}</CS>
+          {<div className={ax([\\"_syaz5scu\\"])}>hello</div>}
+        </CC>;"
+    `);
+  });
+
+  it('should transform css prop using jsx pragma', () => {
+    const result = transform(`
+      /** @jsxRuntime classic */
```

Comment:
What happens with edge cases, should you test those too? i.e.`jsxRuntime automatic` with `@jsx jsx` and `@jsxImportSource` with the runtime specified, etc

Reply:
Build tools (Babel) throw - you can't define both.

Reply:
So would you say that it's not worth having tests for these cases, because it's testing the external babel behaviour?

Reply:
It will throw when defining both - but still good to know what happens in a test (and if its behaviour changes in the future) - I'll add one an expect an exception.

Reply:
Ah, it will throw when transforming the JSX elements with that plugin, however it won't throw in ours. I don't believe it's worth it to double up, what do you think?

Reply:
Yeah, all good to leave it as is